lib/logflare/source/recent_logs_server.ex

```diff
@@ -51,21 +51,6 @@ defmodule Logflare.Source.RecentLogsServer do
   @broadcast_every 500
   @pool_size Application.compile_env(:logflare, Logflare.PubSub)[:pool_size]
 
-  def start_link(%__MODULE__{source_id: source_id} = rls) when is_atom(source_id) do
-    GenServer.start_link(__MODULE__, rls, name: Source.Supervisor.via(__MODULE__, source_id))
-  end
-
-  ## Client
-  @spec init(RLS.t()) :: {:ok, RLS.t(), {:continue, :boot}}
-  def init(rls) do
-    Process.flag(:trap_exit, true)
```

Comment:
Not trapping exits anymore?

Reply:
the `terminate` callback won't fire now.

Reply:
ah mistakenly removed. adding back